packages/mobile/src/components/product/product.tsx

```diff
@@ -109,10 +109,12 @@ const Product: FC<Props> = ({ product, contentContainerStyle }) => {
             ]}
           >
             <Text
+              numberOfLines={3}
```

Comment:
I'd rather go for `numberOfLines={1} ` and `ellipsizeMode="tail"`

Reply:
ok, fixed